app.py

```diff
@@ -46,9 +49,10 @@ def return_login_page():
 
 @app.route('/login', methods=['POST'])
 def process_login():
-    if login_utils.login_verification():
-        flash('flash: You were successfully logged in Eden', "info")
+    password = request.form['pass']
+    username = request.form['name']
+    if login_utils.login_verification(username, password):
+        flash(' You were successfully logged in Eden', category='info')
```

Comment:
What if someone logs in with a different username than Eden? this will be an incorrect statement, right?